ihmc-perception/src/main/java/us/ihmc/perception/ImageSensorPublishThread.java

```diff
@@ -24,19 +28,51 @@ public ImageSensorPublishThread(ROS2Node ros2Node, ImageSensor sensorToPublish,
       publisher = new RawImagePublisher(ros2Node);
    }
 
+   public ImageSensorPublishThread(ROS2Node ros2Node,
+                                   ImageSensor sensorToPublish,
+                                   Map<Integer, ROS2Topic<? extends Packet<?>>> imageKeyToTopicMap,
+                                   double publishFrequencyLimit)
+   {
+      this(ros2Node, sensorToPublish, imageKeyToTopicMap);
+      setFrequencyLimit(publishFrequencyLimit);
+   }
+
+   @Override
+   public ImageSensorPublishThread setFrequencyLimit(double publishFrequencyLimit)
```

Comment:
We can't get rid of the `setFrequencyLimit()` method because it comes from the super class. If we don't want multiple ways of doing things, we gotta remove the parameter from the constructor